docs/tutorials/using-pixy-camera.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Using the CMUcam5 Pixy for LEGO Mindstorms
+group: hardware-extras
+author: "@KWSmit"
+---
+
+* Table of Contents
+{:toc}
+
+## Intro
+
+The CMUcam5 Pixy camera is available in a version designed for
+LEGO Mindstorms. In this tutorial we explain how to use it with ev3dev
+and python.
+
+General information about Pixy for LEGO Mindstorms can be found on
+Charmed Lab's [wiki-page](http://cmucam.org/projects/cmucam5/wiki/LEGO_Wiki).
+
+## What you need
+
+* LEGO Mindstorms EV3.
+* CMUcam5 Pixy for LEGO Mindstorms, connected to the EV3 with the cable
+delivered with your camera. If you have a regular version of CMUcam5 Pixy
+you will need a special Pixy adapter for EV3.
+
+**Note:** the Pixy camera comes with it's own tool: PixyMon. This tool helps
+you to set the signatures (objects you want Pixy to detect). To use this tool
+you need to connect the camera directly to your PC by using a mini USB cable.
+For this you can use the USB cable of your EV3. Beware that when the camera 
+is connected to your PC this way, it's values are not updated! So before 
+starting your script be sure the camera is disconnected from your computer 
+and thus only connected to your EV3.
+
+## Connecting to the camera and reading data
+
+In python use the `Sensor` class to connect to the Pixy camera:
+
+```python
+from ev3dev.ev3 import *
+pixy = Sensor(address=INPUT_1)
+assert pixy.connected, "Error while connecting Pixy camera to port1"
+```
+
+In this statement it is assumed the camera is connected to input port 1.
+It's recommended to use the `assert` statement: when the connection fails
+the program ends with a meaningful error message.
+
+Next set the mode for the camera:
+
+    pixy.mode = 'ALL'
+
+The Pixy camera has the following modes:
+
+* ALL: the camera searches for all signatures you've set for it.
+* SIGn: the camera searches for signature #n (n=1 to 7).
+
+The data which you retreive from the camera depends on the camera mode. You
+can find detailed information on [this page](/docs/sensors/charmed-labs-pixy-cmucam5-for-lego).
+We will explain it to you with some examples.
+
+When mode is set to 'ALL' retreive data the following way:
```

Comment:
I feel like there's something missing here; this phrasing sounds more natural:

```
When the mode is set to 'ALL', you can retrieve data as follows:
```

It may also help to use code formatting (backticks) around mentions of the modes to make it clear that it is a technical string.